ml_metadata/tools/mlmd_bench/fill_types_workload_test.cc

```diff
@@ -30,47 +28,61 @@ limitations under the License.
 namespace ml_metadata {
 namespace {
 
-constexpr int64 kNumOperations = 100;
-constexpr int64 kNumOperationsForMakeUpCases = 200;
-
-// Generates FillTypes Workload where `update` indicates insert or update cases.
-void GenerateFillTypesWorkload(
-    const bool update, const int64 num_operations,
-    std::vector<std::unique_ptr<FillTypes>>& fill_types) {
-  FillTypesConfig fill_artifact_types_config =
-      testing::ParseTextProtoOrDie<FillTypesConfig>(
-          R"(
-            specification: ARTIFACT_TYPE
-            num_properties { minimum: 1 maximum: 10 }
-          )");
-  fill_artifact_types_config.set_update(update);
-  std::unique_ptr<FillTypes> fill_artifact_types = absl::make_unique<FillTypes>(
-      FillTypes(fill_artifact_types_config, num_operations));
-
-  FillTypesConfig fill_execution_types_config =
-      testing::ParseTextProtoOrDie<FillTypesConfig>(
-          R"(
-            specification: EXECUTION_TYPE
-            num_properties { minimum: 1 maximum: 10 }
-          )");
-  fill_execution_types_config.set_update(update);
-  std::unique_ptr<FillTypes> fill_execution_types =
-      absl::make_unique<FillTypes>(
-          FillTypes(fill_execution_types_config, num_operations));
-
-  FillTypesConfig fill_context_types_config =
-      testing::ParseTextProtoOrDie<FillTypesConfig>(
-          R"(
-            specification: CONTEXT_TYPE
-            num_properties { minimum: 1 maximum: 10 }
-          )");
-  fill_context_types_config.set_update(update);
-  std::unique_ptr<FillTypes> fill_context_types = absl::make_unique<FillTypes>(
-      FillTypes(fill_context_types_config, num_operations));
+// Defines a GetTypesResponseType that can be GetArtifactTypesResponse /
+// GetExecutionTypesResponse / GetContextTypesResponse.
+using GetTypesResponseType =
+    absl::variant<GetArtifactTypesResponse, GetExecutionTypesResponse,
+                  GetContextTypesResponse>;
+
+// Gets the number of specific type(artifact type / execution type / context
+// type) inside db, stores the value as `num_types` and updates
+// `get_response`.
+tensorflow::Status GetNumberOfSpecificTypeAndUpdateGetResponse(
+    const FillTypesConfig& fill_types_config, MetadataStore* store,
+    int64& num_types, GetTypesResponseType& get_response) {
```

Comment:
shall this function simply outputs vector<T>& types? 
or unordered_map<int64, T>& types? (id -> type) 

num_types can be dropped by using types.size(), the cases below for get_response can be simplified by looking at the Type

Reply:
Sounds good. I can reuse the function in fill_types_workload.cc for this. Fixed!